ara/cli/host.py

```diff
@@ -48,7 +67,49 @@ def get_parser(self, prog_name):
         return parser
 
     def take_action(self, parsed_args):
-        host = (models.Host.query
-                .filter((models.Host.id == parsed_args.host) |
-                        (models.Host.name == parsed_args.host)).one())
-        return utils.fields_from_object(FIELDS, host)
+        hosts = (models.Host.query
+                 .filter((models.Host.id == parsed_args.host) |
+                         (models.Host.name == parsed_args.host)).one())
+
+        return utils.fields_from_object(FIELDS, hosts)
+
+
+class HostDelete(Command):
```

Comment:
Try it, you'll like it.  Deleting a host *will* delete the corresponding task_results entries.  I'm not sure what you mean by a playbook run being "true".  This is largely here in case you've deleted some playbooks and you have old hosts lying around that in fact no longer existing in your environment and you want to get rid of them.